src/Artsy/Relay/createRelaySSREnvironment.ts

```diff
@@ -16,6 +16,7 @@ import {
   RelayNetworkLayer,
   urlMiddleware,
 } from "react-relay-network-modern/node8"
+import { MetaphysicsVersion } from "Utils/metaphysics"
```

Comment:
Since the network layer we don't use `Utils/metaphysics` anymore. Maybe this file can be deleted?

Reply:
It's still used in two places (time.ts, helpers.ts).